src/login.rs

```diff
@@ -354,13 +361,17 @@ impl Login {
                     // Otherwise just show the error in the relative label.
                     if use_empty_key {
                         self_.content.set_visible_child_name("encryption-key-page");
+                        self_.encryption_key_entry.grab_focus();
```

Comment:
Both the if and else blocks have the same call, so let's remove this redundancy.

Reply:
How did I miss that. Thanks.